Documentation/common-problems.md

```diff
@@ -157,6 +157,22 @@ If `uname -a` shows that you have a kernel version older than `3.15`, you'll nee
 * Disable some Ceph features by starting the [rook toolbox](./toolbox.md) and running `ceph osd crush tunables bobtail`
 * Upgrade your kernel to `3.15` or later.
 
+#### Filesystem Mounting
+
+In the `rook-agent` pod logs, you may see a snippet similar to the following:
+```console
+2017-10-06 18:55:36.462634 I | mount 10.96.186.239:6790,10.97.182.248:6790,10.103.158.61:6790:/: mount: wrong fs type, bad option, bad superblock on 10.96.186.239:6790,10.97.182.248:6790,10.103.158.61:6790:/,
+2017-10-06 18:55:36.462761 I | mount 10.96.186.239:6790,10.97.182.248:6790,10.103.158.61:6790:/:        missing codepage or helper program, or other error
+2017-10-06 18:55:36.462780 I | mount 10.96.186.239:6790,10.97.182.248:6790,10.103.158.61:6790:/:
+2017-10-06 18:55:36.462798 I | mount 10.96.186.239:6790,10.97.182.248:6790,10.103.158.61:6790:/:        In some cases useful info is found in syslog - try
+2017-10-06 18:55:36.462832 I | mount 10.96.186.239:6790,10.97.182.248:6790,10.103.158.61:6790:/:        dmesg | tail or so.
+command mount 10.96.186.239:6790,10.97.182.248:6790,10.103.158.61:6790:/ failed: Failed to complete mount 10.96.186.239:6790,10.97.182.248:6790,10.103.158.61:6790:/: exit status 32
+```
+
+In this case, the agent failed to mount the CephFS because the Kernel does not support the `mds_namespace` option. This is an option available in Kernel 4.7 and newer.
+
+If `uname -a` shows that you have a kernel version older than `4.7`, you'll need to revert back to using the CephFS Kubernetes plugin for consuming the filesystem as describe in this [doc](https://github.com/rook/rook/blob/release-0.5/Documentation/k8s-filesystem.md#deploy-the-application)
```

Comment:
why can't we run with one filesystem only in < 4.7 kernels? Seems odd that we don't work at all.

Reply:
I brought this issue about a month ago and i think the decision was to document it and move forward with ceph-fuse in the future.

We can not pass the `mds_namespace` option but if the users has multiple filesystems created, then it will mount one randomly. 

So we need to think this through carefully.

Reply:
how does the in tree CephFS plugin solve this?

Reply:
agreed, we should be able to just not pass the parameter and it will work for <4.7 kernel. This is running in the agent so he should know what the kernel is, right?

Reply:
I think we should say multiple file systems are not supported for <4.7 kernels. That seems fair. Right now your saying no Cephfs support with agent at all for <4.7 kernels.

Reply:
CephFS kubernetes plugin doenst pass the `mds_namespace`. So it will pick one filesystem in the case where multiple filesystems exist. We can make it behave that way but it might cause issues later when users are trying to attach a particular FS to their pod.

Reply:
In this documentation we are telling them to go outside of Rook and use something else to get that experience.  If we are recommending that then it's probably more smooth to just do it ourselves automatically within Rook.